vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/main/resources/META-INF/resources/frontend/vaadin-spreadsheet/vaadin-spreadsheet-styles.js

```diff
@@ -1055,6 +1054,345 @@ export const spreadsheetStyles = css`
   .clear-filters-button {
     width: 100%;
   }
+
+  /*
+   * Lumo theme variant styles
+   */
+  :host([theme~="lumo"]) {
+    background-color: var(--lumo-base-color);
+  }
+
+  :where(:host([theme~="lumo"])) .v-spreadsheet .sheet .cell {
+    font-family: var(--lumo-font-family);
+    color: var(--lumo-body-text-color);
+    background-color: var(--lumo-base-color);
+    font-size: var(--lump-font-size-s);
+    border-right: 1px solid var(--lumo-contrast-20pct);
+    border-bottom: 1px solid var(--lumo-contrast-20pct);
+    padding: 0 var(--lumo-space-xs);
+  } 
+
+  :host([theme~="lumo"]) .v-spreadsheet {
+    border: 1px solid var(--lumo-contrast-20pct);
+    font-family: var(--lumo-font-family);
+    font-size: var(--lumo-font-size-m);
+    color: var(--lumo-body-text-color);
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .functionbar {
+    background-color: var(--lumo-base-color);
+    border-bottom: 1px solid var(--lumo-contrast-20pct);
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .functionbar .functionfield,
+  .v-spreadsheet .functionbar .addressfield {
```

Comment:
Please note that there are two selectors here (notice the comma at the end of line 1085). The `:host([theme~="lumo"])` prefix is missing on the second selector. There are other similar issues in other selectors below.